Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Umbraco-Interface/index-v7.md

```diff
@@ -0,0 +1,27 @@
+---
+versionFrom: 7.0.0
+---
+
+# Umbraco Interface
+
+## Initial View
+
+Once you have logged in you will enter the Content section of Umbraco. There are several sections within Umbraco that allow you to manage different aspects of your site. The Content section allows you to manage the web pages of your site. Sections may vary from user to user based upon their role. As an editor or writer for instance you may only have access to the Content and Media sections. You can navigate between sections using the black section bar on the left-hand side of the screen. As you rollover one of the sections the name of the section will be revealed. Click a section to navigate to it.
+
+To the right of the section bar is the tree view which shows the structure of your website. To navigate the tree you can click on the black arrows to the left of tree items which will reveal any sub-pages. Clicking on the
+page name will load that page’s content into the large content area on the right of the tree view.
+
+![section tree content.jpg](images/section tree content.jpg)
```

Comment:
This was the image that didn't work. Shouldn't we just get rid of it here as well? 🤔 😅 

```suggestion
```